src/App.jsx

```diff
@@ -1,10 +1,36 @@
-import "./App.css";
+//import "./App.css";
+import { RouterProvider, createBrowserRouter } from "react-router-dom";
+import Navigation from "./components/Navigation";
+import PersonalPage from "./components/PersonalPage/PersonalPage";
+import Home from "./components/Home/Home";
 
 function App() {
   
+  const appRoutes = createBrowserRouter([
+    {
+      path: "/",
+      element: <Home/>
+    },
+    {
+      path: "/home",
+      element: <Home/>
+    },
+    {
+      path: "/",
+      element: <Navigation/>,
+      children: [ 
```

Comment:
Instead of defining a sepearate path for the home component like above, you can do the following:
```
{
    path: "/",
    element: <Home />
}
```